bazel_tools/scala.bzl

```diff
@@ -38,6 +38,8 @@ common_scalacopts = [
     # warn if using deprecated stuff
     "-deprecation",
     "-Xfuture",
+    "-Xsource:2.13",  # fix typechecker bugs
+    "-Ypartial-unification",  # behave more like Scala 2.13
```

Comment:
These comments are not backwards.

Reply:
If you had to comment your comment, probably it's a sign the comments by themselves are not sufficient. Adding more details could make it easier on the readers without having to know about (or remember) this PR. If it's too verbose, a simple pointer to a URL with an understandable explanation is probably good enough.

Reply:
I looked when you were asking about this before, but there was nothing coherent I could link to. I'll just put a description in the comments.

Reply:
e5cb754f850371244684cecb16121839495dc0b0